index.js

```diff
@@ -189,8 +189,10 @@ module.exports = {
       }
 
       self
-        .find(req, { parentId: piece._id, trash: false }, { _id: 1 })
+        .find(req, { parentId: piece.workflowGuid, trash: false }, { _id: 1 })
```

Comment:
changelog must also be updated, and the "release" page in notion

Reply:
I updated the changelog accordingly.